src/main/java/uk/gov/hmcts/ccd/domain/service/validate/DefaultValidateCaseFieldsOperation.java

```diff
@@ -59,78 +48,12 @@
                 + " is not found in case type definition");
         }
         content.setData(fieldProcessorService.processData(content.getData(), caseTypeDefinition, content.getEventId()));
-        caseTypeService.validateData(content.getData(), caseTypeDefinition);
-        validateOrganisationPolicy(caseTypeId, content);
+        caseTypeService.validateData(
+            createValidationContext(content.getData(), caseTypeDefinition)
```

Comment:
we could just use the Validation constructor here maybe

Reply:
DONE